### PR TITLE
Update sentence_translations.json - refractive-index-questions/french

### PR DESCRIPTION
**Pull request by PyStL**
Suggested corrections to french translation up to line 230.

### CODE REVIEW

File: 2023/refractive-index-questions/french/sentence_translations.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
  {
   "input": "And derive the formula describing exactly how much light bends.",
   "model": "nmt",
-  "translatedText": "Et comment trouver la formule décrivant avec précision l'angle de la déviation.",
+  "translatedText": "et dériver la formule décrivant la déviation exacte de la lumière.",
```
Comment (Renelle29) [inline 2023/refractive-index-questions/french/sentence_translations.json:59]: Although "dériver une formule" exists in French, it is not commonly used and has another sense in mathematics that can be misleading, alternatives being "déduire", "déterminer", "exprimer"...

File: 2023/refractive-index-questions/french/sentence_translations.json [hunk 2]
```diff
@@ -101,7 +101,7 @@
  {
   "input": "This particular animation is a little bit messy and hard to follow, so it might be clearer if instead we simply color every point of the plane, such that those points are white near the crests of the wave, and then black away from the crests.",
   "model": "nmt",
-  "translatedText": "Cette animation est un peu brouillonne et difficile à suivre, donc pour gagner en clarté nous colorons simplement chaque point du plan, de telle sorte qu'un point soit blanc près des crêtes de l'onde, et noir à l'écart des crêtes.",
+  "translatedText": "Cette animation en particulier est un peu chargée et difficile à suivre, alors ce serait être plus claire si nous colorions plutôt simplement chaque point du plan, de telle sorte qu'ils soient blancs près des crêtes de la vague, puis noirs à l'écart des crêtes.",
```
Comment (Renelle29) [inline 2023/refractive-index-questions/french/sentence_translations.json:104]: "onde" is the appropriate translation for "wave" in this context, not "vague"

File: 2023/refractive-index-questions/french/sentence_translations.json [hunk 3]
```diff
@@ -182,7 +182,7 @@
  {
   "input": "If it was the case that no bending happened, then because the wavelength is smaller in there, when you look at all those intersection points, they would have to be closer together.",
   "model": "nmt",
-  "translatedText": "Si aucune courbure ne se produisait, alors, comme la longueur d'onde est plus petite dans cet endroit, lorsque vous regardez tous ces points d'intersection, ils seraient plus proches les uns des autres.",
+  "translatedText": "Si c'était le cas qu'il n'y avait pas eu de déviation, alors, comme la longueur d'onde est plus petite à l'intérieur, lorsque vous regardez tous ces points d'intersection, ils devraient être plus proches les uns des autres.",
```
Comment (Renelle29) [inline 2023/refractive-index-questions/french/sentence_translations.json:185]: Overly complex phrasing: "Si c'était le cas qu'il n'y avait pas eu de déviation"
Suggestion: "Si aucune déviation n'avait eu lieu"

File: 2023/refractive-index-questions/french/sentence_translations.json [hunk 4]
```diff
@@ -227,7 +227,7 @@
  {
   "input": "If you do this, what you write down is effectively the same thing as Snell's Law, you just have a tiny bit of added work to relate the relevant angles here, and then to note how the speed and the wavelength all depend on each other.",
   "model": "nmt",
-  "translatedText": "Si vous faites cela, ce que vous obtiendrez est effectivement la même chose que la loi de Snell, vous avez juste un tout petit peu de travail supplémentaire pour relier les angles pertinents ici, puis pour trouver comment la vitesse et la longueur d'onde dépendent l'une de l'autre.",
+  "translatedText": "Si vous faites cela, ce que vous obtienez est effectivement la même chose que la loi de Snell, vous avez juste un tout petit peu de travail supplémentaire pour relier les angles pertinents ici, puis pour remarquez comment la vitesse et la longueur d'onde dépendent l'une de l'autre.",
```
Comment (Renelle29) [inline 2023/refractive-index-questions/french/sentence_translations.json:230]: typo: "obtenez" instead of "obtienez"

**Review by Renelle29 — COMMENTED** [review]
A few suggestions and corrections for wording

File: 2023/refractive-index-questions/french/sentence_translations.json (hunk 1 above)
Comment (PyStL) [reply 2023/refractive-index-questions/french/sentence_translations.json:59]: You're right I hadn't thought about the possible confusion with 'dériver'. I don't think 'déduire' would be appropriate in this context. Of your teo other suggestion, I prefer 'déterminer', but 'obtenir' would also be a good alternative I suppose.
As a more general question regarding this whole translation project, maybe it would be better for conversion to be in the language of the translation being looked at. What do you think?

File: 2023/refractive-index-questions/french/sentence_translations.json (hunk 2 above)
Comment (PyStL) [reply 2023/refractive-index-questions/french/sentence_translations.json:104]: You're right. I did the correction elsewhere, but missed this one.
